qa/rpc-tests/wallet_orchard.py

```diff
@@ -103,7 +103,7 @@ def run_test(self):
         ua3 = addrRes3['address']
 
         recipients = [{"address": ua3, "amount": Decimal('1')}]
-        myopid = self.nodes[2].z_sendmany(ua2, recipients, 1, 0)
+        myopid = self.nodes[2].z_sendmany(ua2, recipients, 1, 0, 'AllowRevealedAmounts')
```

Comment:
`acct2` only has Sapling funds, and `ua3` has a Sapling receiver, so there's no reason why this should require `AllowRevealedAmounts`, as we could create a purely-shielded transaction with Sapling outputs. The reason we don't is because before we even reach the async operation logic, we've preferentially selected the recipient's Orchard receiver. Ensure we have an issue open to fix this bug (asking for a privacy policy we don't actually need).

It occurs to me (likely for a second time) that we could use the privacy policies to optimise note management: if we _could_ create either a fully-private tx or a Sapling-to-Orchard tx, then we could just do the former if no policy is specified, but if `AllowRevealedAmounts` is provided we could use the transaction to optimistically migrate (hiding our transaction in a migration tx we'd otherwise have made anyway).

Reply:
That issue exists; it's https://github.com/zcash/zcash/issues/5801

Reply:
For the record, I don't think it is a good idea to change the behaviour (rather than just to check it) based on the privacy policy.